packages/flutter/lib/src/widgets/image.dart

```diff
@@ -955,7 +956,6 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
 
   @override
   void dispose() {
-    assert(_imageStream != null);
```

Comment:
We might not have ever gotten an _imageStream before being disposed of if we were scrolling fast.

Reply:
How can this happen? Isn't `provider.resolve` in line 1036 always synchronously returning a stream? I also don't see code that changed when we call resolve on the provider...

Reply:
I'll put this back.  This was from a different experiment